Tests/SwerverTests/ActionSpec/NotAllowedActionSpec.swift

```diff
@@ -0,0 +1,23 @@
+import Quick
+import Nimble
+import Swerver
+import Foundation
+
+class NotAllowedActionSpec: QuickSpec {
+    
+    override func spec() {
+        describe("A Not Allowed action") {
+            let listOfAllowedMethods = ["GET", "POST"]
+            
+            let notAllowedAction = NotAllowedAction(methods: listOfAllowedMethods)
+            
+            it("can dispatch to response builder to build a response for a successful OPTIONS request") {
```

Comment:
This should be split into multiple tests. Remember, always ask yourself, can my test be testing less?

In this case, there should be three tests. One test per assertion.

Reply:
will look into that

Reply:
fixed